values-schema.yaml

```diff
@@ -487,9 +493,9 @@ definitions:
       - properties:
           fsGroup:
             description: Supplementary group ID. Volumes that support ownership management are modified to be owned and writable by this ID.
-            type: integer
+            $ref: '#/definitions/linuxUserId'
```

Comment:
why not `unixOwnerId`? I don't think it has something to do with linux specifically, and I see it being used for groups as well

Reply:
I think most of our nodes are Linux based

Reply:
well, we are referring to unix constructs, not linux..